src/components/Editor/CallSites.js

```diff
@@ -142,17 +142,34 @@ class CallSites extends Component {
       });
     }
   }
+  filterCallSites() {
+    const { callSites, breakpoints } = this.props;
+
+    const breakpointLines = new Set();
+    if (breakpoints._root !== undefined) {
```

Comment:
Naming conventions in JS generally treat `_` as an indication that something is private. In this case, the specifics of `_root` are internal to the `breakpoints` object and shouldn't be used directly. The `breakpoints` object is an `Immutable.js` `Map` object, so if you peak at the docs, you've for all sorts of options. https://facebook.github.io/immutable-js/docs/#/Map

In this case, I'd probably do

```
const breakpointLines = new Set(breakpoints.toIndexedSeq().map(bp => bp.location.line));
```
which takes all the values and discards the keys, and then converts each value into its line number.

Reply:
Thanks for pointing me in the right direction for this! I'll definitely take a look and clean this up.